src/newgrf.cpp

```diff
@@ -7008,7 +7000,7 @@ static void ParamSet(ByteReader *buf)
 			break;
 		}
 
-		/* @todo implement */
+		/* not implemented */
```

Comment:
I'd suggest adding a description of why they each is not implemented, just to deter anyone from thinking they need to :)

- Tile refresh offset: Intended to allow support for larger sprites, not necessary for OTTD. (Could arguably implement as a null-action.)
- Snow line height: Better supported by feature 8 property 10h (snow line table). (Could arguably implement by filling the entire snow line table with the given value.)
- Global ID offset: Not necessary since IDs are remapped automatically. (Might be implementable as a null-action?)

I guess nothing of importance actually uses these parameters.

Reply:
> I'd suggest adding a description of why they each is not implemented, just to deter anyone from thinking they need to :)

uhm, it said "todo" for 12 years and nobody was inclined to implement it, why would that be any different now? :p